lua/weapons/weapon_tag.lua

```diff
@@ -129,16 +138,14 @@ function SWEP:PrimaryAttack( right )
 				end
 			end
 			if (target[2] < 50) then
-				TestTagPlayer(self,target[1],self.Owner)
+				self.TestTagPlayer(self,target[1],self.Owner)
```

Comment:
You can call a member function with self:TestTagPlayer (colon instead of dot) and it will implicitly pass self as the first argument

Reply:
That is cool too know. Does a period still work in this case?

Reply:
It works because you passed self in manually, it's just not the "ideal" way to do it